solarforecastarbiter/io/tests/test_io_utils.py

```diff
@@ -216,7 +229,9 @@ def test_adjust_timeseries_for_interval_label_series(label, exp):
         pytest.mark.xfail(strict=True, type=TypeError)]),
     pytest.param(pd.Series([], dtype=float, index=pd.DatetimeIndex([])),
                  marks=[pytest.mark.xfail(strict=True, type=TypeError)]),
-    pytest.param(TEST_DATA, marks=[
+    TEST_DATAFRAME,
+    EMPTY_TIME_DATAFRAME,
```

Comment:
Can you test the serialized output of this explicitly? I'm a little wary about assuming keys are present in the schema for this case